predicators/datasets/generate_atom_trajs_with_vlm.py

```diff
@@ -816,9 +992,12 @@ def create_ground_atom_data_from_generated_demos(
     # Now, given these trajectories, we can query the VLM.
     if vlm is None:
         vlm = utils.create_vlm_by_name(CFG.vlm_model_name)  # pragma: no cover
-    ground_atoms_trajs = _query_vlm_to_generate_ground_atoms_trajs(
-        img_option_trajs, env, train_tasks, known_predicates, all_task_objs,
-        vlm)
+    if CFG.vlm_predicate_vision_api_generate_ground_atoms:
+        generate_func = _generate_ground_atoms_trajs_from_synthesized_predicates
+    else:
+        generate_func = _query_vlm_to_generate_ground_atoms_trajs
```

Comment:
NIT: could you rename `_generate_ground_atoms_from_synthesized_predicates` to `_generate_ground_atoms_with_vlm_oo_code_gen`, and rename `_query_glm_to_generate_ground_atoms_trajs` to `_generate_ground_atoms_with_vlm_pure_visual_preds`?